public/locales/cs/common.json

```diff
@@ -471,7 +471,8 @@
       "TRAINING": "Výcvik",
       "KITEI_MAKOR": "Zdrojové materiály",
       "PUBLICATION": "Publikace",
-      "LELO_MIKUD": "Upraveno - Ne Focus Group"
+      "LELO_MIKUD": "Upraveno - Ne Focus Group",
+      "sources": "Zdroje"
```

Comment:
Is there any special reason for "sources" to be lowercase?

Reply:
Use lowercase for separate sources from true content-type